### DISCUSSION TITLE
Multipolygons in layout optimization

### DISCUSSION THREAD

**Discussion by Rodrig03** [NatLabRockies/floris#581] [Q&A] [answered]
Hello,

Is there a way to perform a layout optimization on a windfarm consisting on 2 or more polygon areas?
This feature would be useful when turbines are not allowed to be installed on a specific area due to regulations. I'm looking to perform the optimization on both areas at the same time and not independently. 

Here is an example of what I mean:
![image](https://user-images.githubusercontent.com/122991766/219344508-7c223bd5-b21a-4434-a0ab-2154cb457246.png)

**Comment by cherifM** [comment, marked as answer]
@Rodrig03 
yes its possible, just make sure you have boundaries similar to :
![image](https://user-images.githubusercontent.com/7900713/220754535-5bf3fd60-4923-4777-bd5d-8bc4a8d335ae.png)
if you connect all polygones , than it works, i have even tried with multiple polygones
best regrads,
cherif
